app/src/main/java/com/example/bookwormadventuresdeluxe2/ProfileFragment.java

```diff
@@ -24,7 +26,7 @@
  *
  * A simple {@link Fragment} subclass.
  */
-public class MyProfileFragment extends Fragment implements FirebaseUserGetSet.UserCallback
+public class ProfileFragment extends Fragment
 {
     private static final String TAG = "MyProfileFragment";
```

Comment:
Is this actually being used anywhere?

Reply:
No, not anymore, I thought it needed it for 
![image](https://user-images.githubusercontent.com/39696855/98053882-db19e600-1df6-11eb-9ea2-57cde4a2e504.png)
but that was not the case